automatic/wps-office-free/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,22 @@
+
+  $version = '10.2.0.5820'
+  $regName = $packageName -replace('\-',' ')
```

Comment:
@AdmiringWorm there are some higher level variables that are defined within chocolatey, which means as a result, they are available in the chocolateyInstall.ps1 file.  I don't think that these should be relied on though, as I don't think they make up part of the public API of Chocolatey.  @ferventcoder thoughts?